crates/sysperf/src/lib.rs

```diff
@@ -39,6 +42,16 @@ pub struct CpuBenchmarkResult {
     pub threads_used: usize,
 }
 
+#[derive(Debug)]
+pub struct SystemInfoResult {
```

Comment:
Could you also return disks and network interfaces?

Reply:
Yeah, sure!